glass/shells.py

```diff
@@ -195,16 +197,39 @@ class RadialWindow(NamedTuple):
     zeff
         Effective redshift of the window.
 
-    Methods
-    -------
-    _replace
-        Create a new instance with changed attribute values.
-
     """
 
     za: NDArray[np.float64]
     wa: NDArray[np.float64]
-    zeff: float = 0
+    zeff: float = dataclasses.field(default=math.nan)
```

Comment:
no need for the overhead

```suggestion
    zeff: float = math.nan
```

Reply:
Sure, I like the explicitness but agree it is superfluous